core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -362,13 +360,15 @@ object ConsumerGroupCommand extends Logging {
       zkUtils.getConsumerGroups().toList
     }
 
-    def deleteGroups() {
+    def deleteGroups(): Map[String, Errors] = {
       if (opts.options.has(opts.groupOpt) && opts.options.has(opts.topicOpt))
         deleteForTopic()
       else if (opts.options.has(opts.groupOpt))
         deleteForGroup()
       else if (opts.options.has(opts.topicOpt))
         deleteAllForTopic()
+
+      Map()
```

Comment:
This is a little odd. Maybe to follow the interface, we should try to catch exceptions and return an error?